mips/pmap.c

```diff
@@ -23,7 +23,6 @@
 #define PTF_SIZE (PTF_ENTRIES * sizeof(pte_t))
 #define PT_ENTRIES (PD_ENTRIES * PTF_ENTRIES)
 #define PT_SIZE (PT_ENTRIES * sizeof(pte_t))
-#define MAX_ASID 255
```

Comment:
No it's not. Asids are platform dependent, as far as I know some MIPS architectures have different max asids. I couldn't spot max asid macro in toolchain so i decided it's best to use ENTRY_HI mask.